TheOracle2/Interactions/SlashCommands/ClockCommand.cs

```diff
@@ -0,0 +1,100 @@
+using Discord.Interactions;
```

Comment:
a "collapsed" version of the previous clock command group. i've retained the command group version in case we want to switch back later, once discord handled subcommand displays more gracefully.